src/components/dropdown/dropdown.e2e.ts

```diff
@@ -22,7 +22,7 @@ describe("calcite-dropdown", () => {
   });
 
   describe("renders", () => {
-    renders(simpleDropdownHTML, { display: "inline-flex" });
+    renders(simpleDropdownHTML, { display: "block" });
```

Comment:
@jcfranco would a change to the default display be considered a breaking change? If not now, then in the future?

Reply:
I'm kind of surprised the screenshot tests don't show any changes when going from `inline-flex` to `block`

Reply:
Well, the screenshot below that has 2 dropdowns next to each other is adjusted for the containing block to apply flex, because when the dropdown is a `block` is will otherwise stack.